### PR TITLE
[EDU-3849] feat: add deploy a service PT

### PR DESCRIPTION
**Pull request by MarianaAguilera**
### Related issue: 

[EDU-3849]

### Changes

add deploy a service PT

### Additional links

n/a.

[EDU-3849]: https://aziontech.atlassian.net/browse/EDU-3849?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/content/docs/pt-br/pages/deploy-jornada/deploy-service.mdx [hunk 1]
```diff
@@ -0,0 +1,96 @@
+---
+title: Implante um serviço
```
Comment (bru-andrade) [inline src/content/docs/pt-br/pages/deploy-jornada/deploy-service.mdx:2]:
```suggestion
title: Como implantar um serviço
```

File: src/content/docs/pt-br/pages/deploy-jornada/deploy-service.mdx [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+- Escolher o `content-type` do recurso. Exemplo: `text` e `shell-script`.
+- Inserir o conteúdo do recurso.
+
+<Button href="#" text="consulte o guia para criar um edge service resource" variant="secondary"></Button>
+
+---
+
+## Passo 3: Gere uma credencial 
+
+Para poder gerenciar edge noces, você deve gerar as credenciais usadas pelo Agente do Orchestrator da Azion.
```
Comment (bru-andrade) [inline src/content/docs/pt-br/pages/deploy-jornada/deploy-service.mdx:52]:
```suggestion
Para poder gerenciar edge nodes, você deve gerar as credenciais usadas pelo Agente do Orchestrator da Azion.
```

File: src/content/docs/pt-br/pages/deploy-jornada/deploy-service.mdx [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+
+Para fazer isso: 
+
+- Escolha uma descrição.
+- Adicione um nome fácil de lembrar.
```
Comment (bru-andrade) [inline src/content/docs/pt-br/pages/deploy-jornada/deploy-service.mdx:57]:
```suggestion
- Dê um nome fácil de lembrar.
```

File: src/content/docs/pt-br/pages/deploy-jornada/deploy-service.mdx [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+
+<Button href="#" text="consulte o guia para instalar o agente do edge orchestrator" variant="secondary"></Button>
+
+---
+
+## Passo 5: Autorize um edge node
+
+Com o Agente do Edge Orchestrator instalado no dispositivo:
+
+- Autorize o edge node.
```
Comment (bru-andrade) [inline src/content/docs/pt-br/pages/deploy-jornada/deploy-service.mdx:80]:
```suggestion
Com o Agente do Edge Orchestrator instalado no dispositivo, autorize o edge node.
```

File: src/content/docs/pt-br/pages/deploy-jornada/deploy-service.mdx [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+<br/>
+<Button href="/pt-br/documentacao/produtos/edge-orchestrator/edge-node/" text="saiba mais sobre edge nodes" variant="secondary"></Button>
+
+---
+
+## Passo 6: Vincule um edge service a um edge node
+
+Agora, com o edge node criado e autorizado, você deve vinculá-lo a um edge service para ser implantado no edge node.
+
+- Crie uma relação entre o edge service e o edge node.
```
Comment (bru-andrade) [inline src/content/docs/pt-br/pages/deploy-jornada/deploy-service.mdx:92]:
```suggestion
Agora, com o edge node criado e autorizado, você deve vinculá-lo a um edge service a ser implantado no edge node.
```

File: src/content/docs/pt-br/pages/deploy-jornada/deploy-service.mdx [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+
+<Button href="#" text="consulte o guia para vincular um edge service a um edge node" variant="secondary"></Button>
+
+Uma vez que essa relação estiver estabelecida, quando uma atualização for acionada nos edge services, ela será implantada em todos os nodes relacionados a este serviço em particular.
```
Comment (bru-andrade) [inline src/content/docs/pt-br/pages/deploy-jornada/deploy-service.mdx:90]:
```suggestion
Uma vez que essa relação estiver estabelecida, quando uma atualização for acionada nos edge services, ela será implantada em todos os nodes relacionados a este serviço em particular.


---

import ContributorList from '~/components/ContributorList.astro'

**Contribuidores** <ContributorList>Contributor</ContributorList>
```

**Review by bru-andrade — APPROVED** [review]
Precisaria alterar em EN tbm :(

**Review by GabrielAzion — APPROVED** [review]

**Pull request merged**